data/strings/strings.txt

```diff
@@ -3660,6 +3740,7 @@
     ja = トイレ
     ko = 화장실
     lt = Tualetas
+    lv = Tualete
```

Comment:
Maybe "Labierīcības"?

Reply:
I'd keep "tualete", as "labierīcības" for a toilet seems to be informal.